specs-go/state.go

```diff
@@ -1,5 +1,20 @@
 package specs
 
+const (
+	// Creating indicates that the container is being created
+	Creating = "creating"
+
+	// Created indicates that the runtime has finished the create operation
+	Created  =  "created"
```

Comment:
I think this file probably needs a `gofmt`. :sweat_smile:

Reply:
Also, given these constants are all within the `specs` namespace, shouldn't they be prefixed with something like `State` or `StateStatus` so it's clear what they're for (as in, `StateStatusCreated`, etc)?

Reply:
Fixed thanks!